packages/election/graphql/resolvers/_mutations/submitElectionBallot.js

```diff
@@ -0,0 +1,22 @@
+const {
+  Roles,
+  ensureSignedIn,
+  Users: {
+    resolveUser,
+    UserNotFoundError
+  }
+} = require('@orbiting/backend-modules-auth')
+
+module.exports = async (_, args = {}, { pgdb, user: me, req }) => {
```

Comment:
Would probably drop or hide "mock" mutation when deploying this to production. No actual technical reason, more like a hunch.